cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamServiceRuntimeModule.java

```diff
@@ -47,8 +53,9 @@ public Module getStandaloneModules() {
       @Override
       protected void configure() {
         bind(StreamFileJanitorService.class).to(LocalStreamFileJanitorService.class).in(Scopes.SINGLETON);
-        bind(StreamWriterSizeCollector.class).to(NoOpStreamWriterSizeManager.class).in(Scopes.SINGLETON);
-        bind(StreamWriterSizeManager.class).to(NoOpStreamWriterSizeManager.class).in(Scopes.SINGLETON);
+        bind(StreamWriterSizeCollector.class).to(NoOpStreamWriterSizeCollector.class).in(Scopes.SINGLETON);
+        bind(StreamWriterSizeFetcher.class).to(InMemoryStreamWriterSizeFetcher.class).in(Scopes.SINGLETON);
```

Comment:
Standalone shouldn't be using the memory fetcher. In Standalone, stream data are persisted to local disk as files.